ProgrammingTask/temp/client.py

```diff
@@ -0,0 +1,183 @@
+import pygame
```

Comment:
temp-directory seems to be old unused code, i think we can safely remove it. Git stores all the previous versions for us.